src/main/integrations/sentry-minidump/minidump-loader.ts

```diff
@@ -69,19 +65,17 @@ export function createMinidumpLoader(
             const twoSecondsAgo = new Date().getTime() - NOT_MODIFIED_MS;
 
             if (stats.mtimeMs < twoSecondsAgo) {
-              const file = await fs.readFile(path);
-              const data = preProcessFile(file);
+              const data = await fs.readFile(path);
+              const parsedMinidump = parseMinidump(data);
```

Comment:
I think we should return an error from `parseMinidump` if we can't parse the minidump and emit debug logging accordingly.